src/main/java/hu/wurfel/refference/school/subject/SubjectDto.java

```diff
@@ -5,6 +5,6 @@
 @Data
 public class SubjectDto {
 	private final Integer id;
-	private final String name;
+	private final String subjectName;
```

Comment:
Ha a subject dto-ban van és a subjecthez tartozik, jobb a `name`.
Ez ugyanígy igaz a subject entityre is.